jupyterhub/app.py

```diff
@@ -568,7 +568,16 @@ def _url_part_changed(self, change):
         urlinfo = urlinfo._replace(netloc=fmt % (self.ip, self.port))
         urlinfo = urlinfo._replace(path=self.base_url)
         bind_url = urlunparse(urlinfo)
+
+        # Warn if both bind_url and ip/port/base_url are set
         if bind_url != self.bind_url:
+            if self.bind_url != "http://:8000" and self.bind_url != "https://:8000":
```

Comment:
I'm not sure why these values are picked out specifically to check against, are they default values?

Reply:
Yes, they are set [here](https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/app.py#L594). Unfortunately, I didn't find a nicer way to check against the default values without hard-coding them :confused:

Reply:
Ah! Will this work perhaps?

```suggestion
            if self.bind_url != self._bind_url_default():
```

Reply:
Yep, it works (I have no idea why I thought it wouldn't :laughing: )
Thanks @consideRatio :heart: